export/export_pipeline/compute_collision_representation.py

```diff
@@ -0,0 +1,427 @@
+                    "-r",
+                    f"{vhacd_args['voxel_resolution']}",
+                    "-e",
+                    f"{vhacd_args['volume_error_percent']}",
+                    "-d",
+                    f"{vhacd_args['max_recursion_depth']}",
+                    "-s",
+                    f"{int(not vhacd_args['disable_shrink_wrap'])}",
+                    "-f",
+                    f"{vhacd_args['fill_mode'].name.lower()}",
+                    "-v",
+                    f"{vhacd_args['max_hull_vert_count']}",
+                    "-a",
+                    f"{int(not vhacd_args['disable_asyn'])}",
+                    "-l",
+                    f"{vhacd_args['min_edge_length']}",
+                    "-p",
+                    f"{int(vhacd_args['split_hull'])}",
+                ],
+                check=True,
+                cwd=mesh_dir,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.STDOUT,
+            )
+            output_str = ret.stdout.decode()
+            print(output_str)
+                    
+            if (ret.returncode != 0) or ('VolumeError' in output_str):
+                print(f"V-HACD failed on {mesh_path}")
+            else: 
+                decomposed = True
+            shutil.move(osp.join(mesh_dir, vhacd_args['outputs'][1]), osp.join(decompose_dir, 'assembled.stl'))
+            os.remove(osp.join(mesh_dir, vhacd_args['outputs'][0]))
+            os.remove(osp.join(mesh_dir, vhacd_args['outputs'][2]))
+            for src_filename in next(os.walk(decompose_dir))[-1]:
+                dst_filename = src_filename.replace(obj_name, '')
+                if dst_filename == '.stl':
+                    continue
+                shutil.move(osp.join(decompose_dir, src_filename), osp.join(decompose_dir, dst_filename))
+        elif acd_args['decompose_method'] == 'none':
+            out_filename = osp.join(decompose_dir, 'assembled.stl')
+            mesh.export(out_filename)
+            out_filename = osp.join(decompose_dir, '000.stl')
+            if params['common']['CONVEX_DECOMPOSITION_EXT'] in mesh_path:
+                shutil.copy(mesh_path, out_filename)
+            else:
+                mesh.export(out_filename)
+            decomposed = True
+        elif acd_args['decompose_method'] == 'skip':
+            decomposed = False
+        else:
+            raise ValueError(f'{mesh_dir} decompose method {acd_args["decompose_method"]} is invalid')
+        
+        print(f"Removing {mesh_path}")
+        os.remove(mesh_path)
+        
+    if not decomposed:  # delete whole directory
+        shutil.rmtree(mesh_dir)
+    return decomposed
+
+
+class CollisionRepresentationComputer(object):
+    def __init__(self, scene_path: str, n_workers: int, rerun: bool, include_objects: str) -> None:
+        self.n_workers = min(max(1, n_workers), mp.cpu_count()-1)
+        self.rerun = rerun
+        self.scene_path = osp.normpath(scene_path)
```

Comment:
Our convention is to use `realpath` over `normpath` unless there is a good reason not to.